src/Microsoft.ML.StandardLearners/Standard/SdcaBinary.cs

```diff
@@ -31,6 +30,12 @@
     "lc",
     "sasdca")]
 
+[assembly: LoadableClass(typeof(SdcaCalibratedBinaryTrainer), typeof(SdcaCalibratedBinaryTrainer.Options),
+    new[] { typeof(SignatureBinaryClassifierTrainer), typeof(SignatureTrainer), typeof(SignatureFeatureScorerTrainer) },
+    SdcaCalibratedBinaryTrainer.UserNameValue,
+    SdcaCalibratedBinaryTrainer.LoadNameValue,
+    "SdcaLogisticRegression")]
```

Comment:
As discussed, would prefer that the command line functionality remain "untyped" thorugh the use of some convenience wrapping class.

Reply:
(Not accessible to API, obviously.)

---
In reply to: [256640856](https://github.com/dotnet/machinelearning/pull/2506#discussion_r256640856) [](ancestors = 256640856)

Reply:
No problem. Please take a look at design concept in `SdcaBinaryTrainerBase<TModelParameters>`'s doc string.

---
In reply to: [256640938](https://github.com/dotnet/machinelearning/pull/2506#discussion_r256640938) [](ancestors = 256640938,256640856)

Reply:
Oh... um huh. OK! Missed that. Is all fixed?

Reply:
entry point and command line behaviors are not changed. :)

---
In reply to: [256660650](https://github.com/dotnet/machinelearning/pull/2506#discussion_r256660650) [](ancestors = 256660650)